src/Membership/AdherentFactory.php

```diff
@@ -65,6 +69,12 @@ public function createFromArray(array $data, bool $enabled = false): Adherent
             isset($data['isAdherent']) ? $data['isAdherent'] : true,
             isset($data['password']) ? $this->encodePassword($data['password']) : null
         );
+
+        if ($syncWithAuth) {
```

Comment:
The Sync should be done after the entity is saved into DB no ?

Reply:
It can be done before, this method is only used by fixtures.